### ISSUE TITLE
Installation Info for Cloud Services, Webhosters and One Click deployments

### ISSUE THREAD

**Issue by mariobehling** [fossasia/open-event-orga-server#1548]
Please ensure that the server runs on cloud services and webhosters and add one-click deployment buttons to set up the system, if possible:
- [ ] Documentation of Vanilla set up on webhosts set up without SSH access https://github.com/fossasia/open-event-orga-server/issues/1547
- [x] AWS documentation
- [x] Digital Ocean
- [x] Google Cloud and setup https://github.com/fossasia/open-event-orga-server/issues/2439
- [ ] Bluemix and setup (#2652)

**Comment by mariobehling** [comment]
@aviaryan You mentioned elsewhere before, that you got the system working on the Google Cloud. Please update the documentation here https://github.com/fossasia/open-event-orga-server/blob/development/docs/INSTALLATION_GOOGLE.md

**Comment by aviaryan** [comment]
@mariobehling I tried one day to deploy the system on Google cloud and mentioned it in the scrum. It didn't work out so I left it. I think I can successfully deploy on AWS so I will try that first. I will come to Google Cloud later. 

> [Scrum] Avi Aryan - Server - 20160802 - False
Tried to deploy Open Event to Google Cloud AppEngine. Sadly it didn't work as I expected. I will try again today but use a Docker instance over the app.
Issue - https://github.com/fossasia/open-event-orga-server/issues/1548
